File: Practice/17/Python/17.py
yes = []
no = []
r,b = 0, 0
for i in range(37):
  no.append(1)
  yes.append(0)
while 1:
  n = int(input())
  if n < 0: break
  if n == 1 or n == 3 or n == 5 or n == 7 or n == 9 or n == 12 or n == 14 or n == 16 or n == 18 or n == 19 or n == 21 or n == 23 or n == 25 or n == 27 or n == 30 or n == 32 or n == 34 or n == 36:
    r+=1
  elif n!=0:
    b+=1
  yes[n]+=1
  no[n] = 0
  maxi = yes[0]
  for i in range (37):
    if yes[i] > maxi: maxi = yes[i]
  for i in range (37):
    if yes[i] == maxi: print(i, end = ' ')
  print('\n')
  for i in range (37):
    if no[i] == 1:
      print(i, end = ' ')
  print('\n', r,' ', b,'\n\n', sep='')
  no[n] = 0

File: Practice/17/C++/17/17/17.cpp
#include <iostream>
using namespace std;
int main() {
	long long yes[37], no[37], r = 0, b = 0, n = 0;
	for (int i = 0; i < 37; i++) {
		no[i] = 1;
		yes[i] = 0;
	}
	while (1) {
		cin >> n;
		if (n < 0) break;
		if (n == 1 || n == 3 || n == 5 || n == 7 || n == 9 || n == 12 || n == 14 || n == 16 || n == 18 || n == 19 || n == 21 || n == 23 || n == 25 || n == 27 || n == 30 || n == 32 || n == 34 || n == 36)
			r++;
		else if (n != 0)
			b++;
		yes[n]++;
		no[n] = 0;
		int maxi = yes[0];
		for (int i = 0; i < 37; i++)
			if (yes[i] > maxi)
				maxi = yes[i];
		for (int i = 0; i < 37; i++)
			if (yes[i] == maxi)
				cout << i << " ";
		cout << "\n";
		for (int i = 0; i < 37; i++)
			if (no[i] == 1)
				cout << i << " ";
		cout << "\n" << r << " " << b << "\n\n";
		no[n] = 0;
	}
}